crates/client/block_production/src/lib.rs

```diff
@@ -77,6 +77,15 @@ pub enum Error {
     #[error("State diff error when continuing the pending block: {0:#}")]
     PendingStateDiff(#[from] StateDiffToStateMapError),
 }
+
+struct ContinueBlockResult {
```

Comment:
could you document this struct for rust doc?

Reply:
added